docs/developer.md

```diff
@@ -0,0 +1,64 @@
+# Mobile App Developer's Guide
+
+This guide outlines the best practices to follow when you’re building a mobile app for the Province of B.C. 
+
+
+## Digital Code of Practice
+
+All apps developed for the Province of B.C. must adhere to the [Digital Code of Practice](https://digital.gov.bc.ca/policies-standards/dcop/) (DCoP). It provides a comprehensive set of guidelines and principles to guide your work. The DCoP emphasizes user-centricity, accessibility, security, and efficiency. Adhering to it will help you achieve digital excellence and improve citizen experience.
+
+## Code location
```

Comment:
make this an H3, if it's related to DCOP

Reply:
restructured the document to make better use of header levels